.github/workflows/elm_tests.yml

```diff
@@ -71,6 +71,17 @@ jobs:
       - name: Run elm-review on source code
         run: npx elm-review
 
+      - name: Checkout the website-copy-repo
+        uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791
+        with:
+          repository: exercism/website-copy
+          path: website-copy
+
+      - name: Check comment paths exist
+        run: |
+          ./bin/build.sh
+          ./bin/check_website-copy_paths.sh website-copy
```

Comment:
TODO: "Check comment paths exist" should be in a separate job since we expect it to fail often. It requires all of the installing steps above so I need to refactor that first.

Reply:
It seemed quite complicated to have shared steps between jobs, so I settled for a little bit of repetition in the end.